pkg/kubelet/server.go

```diff
@@ -346,6 +352,18 @@ func (s *Server) InstallDebuggingHandlers() {
 	s.restfulCont.Add(ws)
 }
 
+// InstallApiHandlers registers the HTTP request patterns that serve the kubelet versioned api.
+func (s *Server) InstallApiHandlers() {
+	ws := new(restful.WebService).
+		Path("/api/v1/").
+		Consumes(restful.MIME_JSON).
+		Produces(restful.MIME_JSON)
+
+	ws.Route(ws.GET("rawMetrics").To(handlers.RawMetricsHandler(s.host, s.host.GetStatusManager())))
```

Comment:
Personally I'd prefer a hierarchy like (dropping `experimental` prefixes, etc:

```
/metrics/nodes[/nodeId]
/metrics/pods
/namespaces/<namespace>/metrics/pods[/podId]
```

Seems to feel more RESTy to me & reads nicer. You could add `/raw` or whatever else you want if you want to get more granular?